Directory.Build.props

```diff
@@ -498,6 +498,7 @@
   <PropertyGroup>
     <CustomBeforeNoTargets>$(RepositoryEngineeringDir)NoTargetsSdk.BeforeTargets.targets</CustomBeforeNoTargets>
     <CustomAfterTraversalTargets>$(RepositoryEngineeringDir)TraversalSdk.AfterTargets.targets</CustomAfterTraversalTargets>
+    <AfterMicrosoftNetSdkProps>$(AfterMicrosoftNetSdkProps);$(RepositoryEngineeringDir)NetSdk.AfterProps.props</AfterMicrosoftNetSdkProps>
```

Comment:
To be honest, I don't like using the `AfterMicrosoftNetSdkProps` hook for this purpose. The `BuildHostTools` property is only read by items which evaluate after properties. I think it should be fine to set this property in Directory.Build.targets. Do you see a strong requirement to have the property available before a project file gets evaluated?

Reply:
I was going to try to hook this in with #111512 for `BuildHostILTools`. I'll prototype locally and see if I can come up with a better way to do that.

Reply:
If there isn't a better way to solve this then this is totally fine.